### ISSUE TITLE
Custom Icons with React Native

### ISSUE THREAD

**Issue by franbusleiman** [pavel-corsaghin/react-native-leaflet#37]
Hi, does anyone found a solution for creating custom icons using React Native Components? Or at least using html components. Thank you!

**Comment by It-samuel** [comment]
you can equally download the PNG of the icon and use 
hope i got your question right

**Comment by franbusleiman** [comment]
the problem is that I don't want to use a png as the icon, because I want to modify it with css before rendering.

**Comment by It-samuel** [comment]
you can modify the png with css before rendering or are you not using react native

**Comment by franbusleiman** [comment]
yes, I'm using react-native. Could you provide me some info on how to do it? Or some links? Thank you in advance.
